src/api/Tron.js

```diff
@@ -0,0 +1,541 @@
+// @flow
+import { BigNumber } from "bignumber.js";
+import type {
+  Transaction,
+  TrongridTxInfo,
+  SendTransactionData,
+  SendTransactionDataSuccess,
+  SmartContractTransactionData,
+  FreezeTransactionData,
+  UnfreezeTransactionData,
+  NetworkInfo,
+  BandwidthInfo,
+  SuperRepresentative,
+  SuperRepresentativeData,
+  TronResources
+} from "../families/tron/types";
+import type { Account, SubAccount, Operation } from "../types";
+import {
+  decode58Check,
+  encode58Check,
+  abiEncodeTrc20Transfer,
+  formatTrongridTxResponse,
+  hexToAscii
+} from "../families/tron/utils";
+import { log } from "@ledgerhq/logs";
+import network from "../network";
+import { retry } from "../promise";
+import { makeLRUCache } from "../cache";
+import { getEnv } from "../env";
+import get from "lodash/get";
+import drop from "lodash/drop";
+import sumBy from "lodash/sumBy";
+import take from "lodash/take";
+
+const baseApiUrl: string = getEnv("API_TRONGRID_PROXY");
+
+async function post(url: string, body: Object) {
+  try {
+    const { data } = await network({
+      method: "POST",
+      url,
+      data: body
+    });
+
+    log("http", url);
```

Comment:
please remove the log. it's going to duplicate our logs. there is a bigger proposal to come as part of https://github.com/LedgerHQ/ledger-live-common/pull/501 too